meshroom/ui/qml/Viewer/Viewer2D.qml

```diff
@@ -621,6 +698,52 @@ FocusScope {
                             checkable: true
                             checked: false
                             enabled: root.aliceVisionPluginAvailable
+                            onCheckedChanged : {
+                                if((displayLensDistortionViewer.checked || displayPanoramaViewer.checked) && checked){
+                                    displayLensDistortionViewer.checked = false;
+                                    displayPanoramaViewer.checked = false;
+                                }
+                            }
+                        }
+                        MaterialToolButton {
+                            id: displayLensDistortionViewer
+                            property var activeNode: root.aliceVisionPluginAvailable ? _reconstruction.activeNodes.get('sfm').node : null
+                            property bool isComputed: activeNode && activeNode.isComputed
+
+                            ToolTip.text: "Lens Distortion Viewer"
+                            text: MaterialIcons.panorama_horizontal
+                            font.pointSize: 16
+                            padding: 0
+                            Layout.minimumWidth: 0
+                            checkable: true
+                            checked: false
+                            enabled: activeNode && isComputed
+                            onCheckedChanged : {
+                                if((displayHDR.checked || displayPanoramaViewer.checked) && checked){
+                                    displayHDR.checked = false;
+                                    displayPanoramaViewer.checked = false;
+                                }
+                            }
+                        }
+                        MaterialToolButton {
+                            id: displayPanoramaViewer
+                            property var activeNode: root.aliceVisionPluginAvailable ? _reconstruction.activeNodes.get('SfMTransform').node : null
+                            property bool isComputed: activeNode && activeNode.isComputed
```

Comment:
We don't need the SfMTransform to be computed. We only need to have the node of the `activeNode.input` to be computed.